diff --git a/BUILD/pyenv-2.3.14/libexec/pyenv b/home/mdegenaro/pyenv
index 35161cf..75e3efb 100755
--- a/libexec/pyenv
+++ b/libexec/pyenv
@@ -77,7 +77,7 @@ export PYENV_DIR
 shopt -s nullglob

 bin_path="$(abs_dirname "$0")"
-for plugin_bin in "${bin_path%/*}"/plugins/*/bin; do
+for plugin_bin in "${bin_path%/*}"/plugins/*/bin /usr/share/pyenv/plugins/*/bin; do
   PATH="${plugin_bin}:${PATH}"
 done
 # PYENV_ROOT can be set to anything, so it may happen to be equal to the base path above,